libs/fairness/src/lib/util/calculateFairnessMetric.test.ts

```diff
@@ -0,0 +1,197 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { calculateFairnessMetric } from "./calculateFairnessMetric";
+import { FairnessModes } from "./FairnessMetrics";
+
+const closeTo = (expected: number, precision = 2) => ({
+  asymmetricMatch: (actual: number) =>
+    Math.abs(expected - actual) < Math.pow(10, -precision) / 2
+});
+describe("calculateFairnessMetric", () => {
+  it("should return NaN for no values", () => {
+    const mockValue = {
+      bins: [],
+      global: 0.5
+    };
+    const result = calculateFairnessMetric(mockValue, FairnessModes.Difference);
+    const expectedResult = {
+      overall: Number.NaN
+    };
+    expect(result).toEqual(expectedResult);
+  });
+  it("should calculate difference without binBounds correctly", () => {
+    const mockValue = {
+      bins: [0.5, 0.2],
+      global: 0.5
+    };
+    const result = calculateFairnessMetric(mockValue, FairnessModes.Difference);
+    const expectedResult = {
+      overall: 0.3
+    };
+    expect(result).toEqual(expectedResult);
+  });
+  it("should calculate difference (with overlap) correctly", () => {
```

Comment:
Great to see these tests. Since Miro's proposed formulae make assumptions that `x1 < y1`, can you repeat the tests with the order of bins reversed, making sure that you get the same overall answers?